docs/index.md

```diff
@@ -65,15 +65,25 @@ The challenge hint articles  are available on the freeCodeCamp forum as hint top
 
 ### Can I translate freeCodeCamp's curriculum?
 
-At the minute we are translating the curriculum to Chinese _only_.
+Thanks for your interest in translating the curriculum.
 
-While we do intend to make the curriculum available in more languages, right now we do not have a timeline for this.
+Currently, we are translating the curriculum to support only the Chinese language.
```

Comment:
```suggestion
Currently, we are only translating the curriculum into Chinese.
```
The wording suggested (to me) that we'd only have a Chinese curriculum and we'd no longer support English.